adal/devApps/XFormsApp/SecondPage.cs

```diff
@@ -158,15 +212,22 @@ void LogCallback(LogLevel level, string message, bool containsPii)
             LoggerCallbackHandler.LogCallback = LogCallback;
         }
 
-        private async void AcquireTokenSilentButton_Clicked(object sender, EventArgs e)
+        async void AcquireTokenButton_Clicked(object sender, EventArgs e)
```

Comment:
1. Is it intentioanal that you made this internal?
2. Consider keeping visibility modifiers explicit because it makes the code easier to read.